python/runtime/cudaq/spin/py_spin_op.cpp

```diff
@@ -103,9 +103,14 @@ void bindSpinOperator(py::module &mod) {
       .def("distribute_terms", &cudaq::spin_op::distribute_terms,
            "Return a list of `SpinOperator` representing a distribution of the "
            "terms in this `SpinOperator` into equally sized chunks.")
-      .def_static("random", &cudaq::spin_op::random,
-                  "Return a random spin_op on the given number of qubits and "
-                  "composed of the given number of terms.")
+      .def_static(
+          "random", &cudaq::spin_op::random,
+          "Return a random SpinOperator on the given number of qubits and "
+          "composed of the given number of terms.")
+      .def_static("from_word", &cudaq::spin_op::from_word,
```

Comment:
I believe`from_string` would be more uniform naming with the `to_string` method

Reply:
`from_string` is more general than I wanted to imply here. `from_string` could be something like `XY + YX`, but here I just want single Pauli words.